packages/chat-client/src/types/engine.ts

```diff
@@ -83,16 +83,19 @@ export abstract class IChatEngine {
   // ---------- Protected Relay Event Methods ----------------------------------- //
 
   protected abstract onRelayEventRequest(
-    event: EngineTypes.EventCallback<JsonRpcRequest>
+    event: EngineTypes.EventCallback<JsonRpcRequest>,
+    publishedAt: number
```

Comment:
Curious: how come you didn't extend `EventCallback` to account for `publishedAt` here? 

That's the route I took on the push side: https://github.com/WalletConnect/push-client-js/commit/de9d8e62d6f08025b606e810666efb4a5666a24d#diff-cdbc160ba7b6e0945ec9bcdfc19df1e269c6718245d7bf30122fda5e7bde5d47R24

Reply:
Since it was only really used in one place, I decided not to change something every engine event depends on, if there ever is a need for it I can go that route

Reply:
Fair, thanks for context 👍